xml/depl_multiple_networks.xml

```diff
@@ -0,0 +1,433 @@
+    <para>
+        From the node verify that SSH is working by opening an SSH session to 
+the VM.
+</para>
+</step>
+</procedure> 
+</sect1>
+
+<sect1 xml:id="sec.automatenet">
+    <title>Automating Network Setup</title>
+    <para>
+        Ideally, we need to be able to setup a second external network using 
+&crow;. This can be done as manual step, but this is error-prone.
+</para>
+
+<para>
+We should be able to handle the configuration file of the OVS agent that is on 
+the network node in a separate way. The OVS agent runs on every compute node, 
+and with the current configuration the compute node is not connected to the 
+external network.
+</para>
+<para>
+    If the parameter <parameter>bridge_mappings</parameter> is set, and the 
+agent doesn't find the bridges specified in the mappings, it will exit. That is
+why we need one configuration file for the agent running on the network node 
+with <parameter>bridge_mappings</parameter> specified, and one configuration 
+for the agent running on the compute node without 
+<parameter>bridge_mappings</parameter> . (Unless you embrace the 
+&o_netw; Distributed Virtual Router (DVR) and set a connection to the 
+external network on every compute node.)
+</para>
+<tip>
+    <title>Setting up DVR</title>
+    <para>
+For setting up DVR apart from having a connection to the external network 
+on every compute node, it seems that only configuration changes are required, 
+and the support of l2 POP and VXLAN.
+</para>
+</tip>
+
+<para>
+To automate creating external networks you need  
+<link xlink:href="https://github.com/SUSE-Cloud/automation">
+    <citetitle>mkcloud</citetitle></link>
+Start by following these steps to modify the &o_netw; &barcl;.
+</para>
+
+<procedure xml:id="pro.automate-net">
+    <step>
+    <para>
+        First, run the following script.
+    </para>
+    <screen>
+sudo ./runtest.c18.rossella_s prepare setupadmin addupdaterepo runupdate \ 
+ prepareinstcrowbar
+    </screen>
+</step>
+
+<step>
+<para>
+    Then add the following lines to 
+    <filename>/etc/crowbar/network.json</filename>, substituting your own VLAN 
+ID and network values.
+</para>
+<screen>
+        "public2": {
+          "conduit": "intf1",
+          "vlan": <replaceable>600</replaceable>,
+          "use_vlan": true,
+          "add_bridge": false,
+          "subnet": "<replaceable>192.168.135.128</replaceable>",
+          "netmask": "<replaceable>255.255.255.128</replaceable>",
+          "broadcast": "<replaceable>192.168.135.255</replaceable>",
+          "ranges": {
+            "host": { "start": "<replaceable>192.168.135.129</replaceable>", 
+               "end": "<replaceable>192.168.135.254</replaceable>" }
+          }
+    },
+</screen>
+</step>
+
+
+<step>
+    <para>
+        Now run <command>./runtest.c18.rossella_s instcrowbar</command>
+        </para>
+    </step>
+    
+    <step>
+    <para>
+        In your <command>mkcloud</command> environment add a new VLAN. In the 
```

Comment:
lines 375-383 are not needed. I needed those steps to fake a second public network, the user should have a real one